src/Header/Header.css

```diff
@@ -1,3 +1,70 @@
-.div {
-  background-color: blue;
+.head {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.25);
+  z-index: 1;
 }
+
+.greeting-row {
+  margin-left: 335px;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.greeting {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.button-row {
+  padding: 5px 10px;
+  width: 335px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.share {
+  border: 0;
+  box-shadow: none;
+  padding: 8px 24px;
+  background: #71B2FF;
+  border-radius: 5px;
+}
+
+.text1 {
```

Comment:
I would probably give this a more descriptive name.
Also it looks like 3 of the properties set on here are also applied to `.text2`, it might be worth including 2 classes on this element and removing the duplicated properties.